modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -855,6 +862,8 @@ private RaftGroupOptions groupOptionsForPartition(boolean isVolatileStorage, Sna
 
         raftGroupOptions.commandsMarshaller(raftCommandsMarshaller);
 
+        partitionRaftConfigurator.configure(raftGroupOptions);
```

Comment:
Only Loza do this in code above. Why do you think it ought to be done explicitly before Loza is called?